.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**What this PR does / why we need it**:
+
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
+Fixes #
+
+**Please check the following list**:
+- [ ]  Does the affected code have corresponding tests, e.g. unit test, E2E test?
+- [ ]  Does this change require a documentation update?
```

Comment:
This is also not applicable since tspclient-go does not have a separated doc site.